app/javascript/packs/application/redux/tours/index.js

```diff
@@ -24,20 +27,27 @@ export const reducer = (state = INITIAL_STATE, action) => {
     case UPDATE_TOUR:
     case LIST_TOURS:
     case GET_TOUR:
+    case DELETE_TOUR:
       return { ...state, isFetching: true };
     case CREATE_TOUR_SUCCESS:
     case UPDATE_TOUR_SUCCESS:
-    case LIST_TOURS_SUCCESS:
     case GET_TOUR_SUCCESS:
+    case DELETE_TOUR_SUCCESS:
```

Comment:
The reducer here should delete the tour from state. There shouldn't be any data contained in the action.

```javascript
return {
  ...state,
  isFetching: false,
  data: omit(state.data, action.tourId)
}
```

In the example, `omit` comes from `lodash`.